updated about mac activity

Agrim1234 opened the pull request:
fixes #82 
contributors changed and made like ard 
more by mac section added
and linkedin, website and github link added

Agrim1234 committed:
first commit

Agrim1234 committed:
first commit

Agrim1234 committed:
second commit

Agrim1234 committed:
third

Agrim1234 committed:
third

Agrim1234 committed:
Merge remote-tracking branch 'upstream/dev' into dev

Agrim1234 committed:
fourth

Agrim1234 committed:
fourth

Agrim1234 committed:
fifth

Agrim1234 committed:
fifth

Rushi98 commented:
resolve the conflicts 

start by doing
```bash
git pull --rebase https://github.com/MobileApplicationsClub/DoJMA dev
```

YashxD committed:
Issues #88, #80, #78, #74 (#92)

* Bottom NavBar Added and Slideshow removed.

* Bottom NavBar Added and Slideshow removed.

* AndroidClient:
Bottom NavBar added.
Overflow Menu added.
Favouries and Issues Fragments updated to funtionality.

tanmaydixit committed:
corrections and dependency upgraded

Rushi98 committed:
refactoring

Agrim1234 committed:
Resolved merge conflicts

Agrim1234 committed:
Merge branch 'dev' of https://github.com/Agrim1234/dojma into dev

Agrim1234 commented:
conflicts resolved
